### ISSUE TITLE
Question: does this project only works on India 

### ISSUE THREAD

**Issue by nouralddin-abdullah** [Piyushhbhutoria/whatsappWebAPI#28]
Question: does this project only works on India

**Comment by Piyushhbhutoria** [comment]
No, this works for all international numbers you must always use country code attached to the number ex recipient is 123456789 and country code is +1 then use 1123456789.
